README.md

```diff
@@ -305,6 +305,12 @@ You can still override the type if you want to use a different one (e.g., if the
 y q 42 least_squares:my_namespace.MySpecialType '[[10, 1], [20, 2]]'
 ```
 
+If your service call requires multiple arguments, specify them as follows:
+
+```bash
+yakut call 125 435:uavcan.node.ExecuteCommand '{command: 65533, parameter: "firmware.bin"}'
+```
```

Comment:
Could you please check that this is also valid (by design it should be) while you're at it? Maybe it also should be added to the help output.

```suggestion
If your service call requires multiple arguments, specify them as follows:

```bash
yakut call 125 435:uavcan.node.ExecuteCommand '{command: 65533, parameter: "firmware.bin"}'
yakut call 125 435:uavcan.node.ExecuteCommand '[65533, "firmware.bin"]'  # Shorter but position-sensitive
```
```

Reply:
Yes it works.